orchestrator/domain/base.py

```diff
@@ -846,10 +849,7 @@ def _set_instance_domain_model_attrs(
         subscription_instance.depends_on_block_relations = depends_on_block_relations
 
     def save(
-        self,
-        *,
-        subscription_id: UUID,
-        status: SubscriptionLifecycle,
+        self, *, subscription_id: UUID, status: SubscriptionLifecycle, skip_relation_check: bool = False
```

Comment:
Why do we need a `skip_relation_check` parameter

Reply:
As discussed during a standup last week: the code in workflows was safe: but we didn't honor the safe way in some unit tests; that's now visible. We can also fix this by rewriting some of the text fixtures that deliver complexer subscriptions (for unit tests that test functionality against there complexer subscriptions): You would need to add the necessary lifecycle options there (so more boilerplate): you can find the places by searching in the gitlab MR that accompanies this ticket. Both solutions were discusses: and this was de chosen approach.